kinto/views/buckets.py

```diff
@@ -0,0 +1,39 @@
+from cliquet import resource
+
+from kinto.views import NameGenerator
+
+
+buckets_options = {
+    'collection_methods': ('GET',)
+}
+
+
+@resource.register(name="bucket", **buckets_options)
+class Bucket(resource.BaseResource):
+
+    def __init__(self, *args, **kwargs):
+        super(Bucket, self).__init__(*args, **kwargs)
```

Comment:
When reading this, I see that we're inheriting a bunch of methods. Should we instead add hooks in the `BaseResource` class?